bundles/binding/org.openhab.binding.withings/README.md

```diff
@@ -1,89 +1,123 @@
-
-The OAuth configuration must be stored in the file `services/withings-oauth.cfg`.
+The Withings Binding uses a default application registration to request an OAuth
+token. The application belongs to the binding developer. If the user wants to
+use his own application, the binding can be configured to use another OAuth
+consumer key and consumer secret. Please read the [Withings documentation](http://oauth.withings.com/en/partner/dashboard)
+on how to register an application. After the application is created, Withings
+will generate a consumer key and secret. The redirect URL must be configured,
+too.
 
-### Multiple accounts
-
-If one owns multiple withings accounts these accounts must be configured specifically. In addition to the above mentioned `services/withings.cfg` configuration, an `accountId` can be specified.
-
-```
-thomas.token=74c0e77021ef5be1ec8dcb4dd88c1xckusadwe92f9541c86799dcbbfcb8fc8b236
-thomas.tokensecret=25f1098209xns511711b3287288f90740ff45532cef91658c5043db0b0e0c851c
-peter.token=74c0e77021ef5be1ec8dcb4dd88c1xckusadwe92f9541c86799dcbbfcb8fc8b236
-peter.tokensecret=25f1098209xns511711b3287288f90740ff45532cef91658c5043db0b0e0c851c
-```
 
 ## Item Configuration
 
-To bind a measure value to an item, the measure type has to be defined in the item binding config. Withings data can be bound to `Number` items only. The syntax for a Withings binding is 
+To bind a measure value to an item, the measure type has to be defined in the
+item binding config. Withings data can be bound to `Number` items only. The
+syntax for a Withings item binding is:
 
-```
+````
 withings="<measure type>"
-``` 
+````
 
-The following table shows the measure types and units, that are supported by the binding:
+The following table shows the measure types and units that are supported by the binding:
 
-| Measure type | Binding Config | Unit |
-|--------------|----------------|------|
-| Weight       | weight         | kg   |
-| Height       | height         | meter |
-| Fat Free Mass | fat_free_mass | kg   |
-| Fat Ratio    | fat_ratio      | %    |
+| Measure Type    | Binding Config | Unit |
+|-----------------|----------------|------|
+| Weight          | weight         | kg   |
+| Height          | height         | meter |
+| Fat Free Mass   | fat_free_mass | kg   |
+| Fat Ratio       | fat_ratio      | %    |
 | Fat Mass Weight | fat_mass_weight | kg |
 | Diastolic Blood Pressure | diastolic_blood_pressure | mmHg |
 | Systolic Blood Pressure | systolic_blood_pressure | mmHg |
-| Heart Pulse | heart_pulse | bpm |
+| Heart Pulse     | heart_pulse | bpm |
 
 
 ## Examples
 
-The following snippet shows some sample bindings:
+### Examples of multiple account configuration

```

Comment:
Could we add a single-user example here too, pls?

Reply:
My understanding is no special setup is needed for a single user.